airflow/providers/common/sql/hooks/sql.py

```diff
@@ -166,6 +175,7 @@ def __init__(self, *args, schema: str | None = None, log_sql: bool = True, **kwa
         # Hook deriving from the DBApiHook to still have access to the field in its constructor
         self.__schema = schema
         self.log_sql = log_sql
+        self._fast_executemany = fast_executemany
```

Comment:
Should this also be set on the class instead? (`supports_fast_executemany`?) Or does this depend on some information that’s only available at runtime?

Reply:
I removed the self._fast_executemany property, as I tested without it using the executemany it had the same performance.